Script for automated diff

annaghi opened the pull request:
You can compare the elm-css generated file with the original raw file found [here](https://raw.githubusercontent.com/necolas/normalize.css/master/normalize.css).

**README.md**
* update with a Testing section

**Stylesheets.elm**
* move from `examples/` to `tests/`

**tests/**
* `run-tests.sh` the script for comparing the files
* everything the script needs (`package.json`, `elm-package.json`)


The elm-css generated `normalize.css` file will be placed in the **tests/** folder. The `run-tests.sh` script will not remove that.
All the other generated files will be placed in the temporary **tests/css/** folder.

Commit:
Update for comparing the CSS files

scottcorgan commented:
💯  This is a great idea. It would be ideal if all relics of running the tests would be removed after the tests run.

Also, we should probably keep the examples directory. It's kind of Elm-ish practice to have a working example for people to play with.

Commit:
Move Stylesheets.elm back to examples directory

Commit:
Update the script

annaghi commented:
Thanks! :smile: 

Yes, I was hesitating which would be the best location of the `Stylesheets.elm`. Moved back to its original directory, and modified the script too. Let me know if anything remained to be done.
